Dockerfile

```diff
@@ -24,9 +24,9 @@ RUN mv android-sdk-linux /usr/local/android-sdk
 RUN rm android-sdk_r24.4.1-linux.tgz
 
 # Install Android tools
-ENV ANDROID_SUPPORT_VERSION 24.0.0
+ENV ANDROID_SUPPORT_VERSION 24.1.1
 RUN (while :; do echo 'y'; sleep 2; done) | /usr/local/android-sdk/tools/android update sdk --filter extra-android-support,extra-android-m2repository,platform-tool --no-ui -a
-RUN (while :; do echo 'y'; sleep 2; done) | /usr/local/android-sdk/tools/android update sdk --filter build-tools-22.0.1,build-tools-23.0.1,build-tools-23.0.2,build-tools-23.0.3,build-tools-21.1.2,build-tools-24.0.0,build-tools-21.0.3,android-21,android-22,android-15,android-23,android-24 --no-ui -a
+RUN (while :; do echo 'y'; sleep 2; done) | /usr/local/android-sdk/tools/android update sdk --filter build-tools-22.0.1,build-tools-23.0.3,build-tools-21.1.2,build-tools-24.0.1,build-tools-24.0.0,build-tools-21.0.3,android-21,android-22,android-15,android-23,android-24 --no-ui -a
```

Comment:
Maybe a typo? You changed build-tools-23.0.2 to an older version build-tools-21.1.2 ?